### DISCUSSION TITLE
Unable to self-host with Windows Server 2016: No matching manifest

### DISCUSSION THREAD

**Discussion by enpro-github** [Budibase/budibase#3219] [Help]
I'm attempting to use Windows Server 2016 to self-host Budibase but when I run the `budi hosting --start` command I receive the error: `Pulling minio-service (minio/minio:)...
latest: Pulling from minio/minio
no matching manifest for windows/amd64 10.0.14393 in the manifest list entries`

Is there a setting or version of minio I can indicate to get one that is compatible with Windows Server 2016? Or is there any other solution? Thank you

**Comment by mjashanks** [comment]
Hi @enpro-github 

Minio doesn't have a windows image, but works using windows subsystem for linux. However, it seems that WSL is not available on Windows Server 2016 - https://docs.microsoft.com/en-us/windows/wsl/install-on-server

Minio can be installed by executable: https://docs.min.io/docs/minio-quickstart-guide.html

Then you would need to follow these setup instructions: https://docs.min.io/docs/minio-quickstart-guide.html, but modify the `docker-compose.yaml` to 
1. Exclude any mention of "minio-service".
2. Update the `MINIO_URL` to be the URL of your running minio instance.

I haven't tried this, so can't guarantee success - but that's the general idea.

If you want to give Budibase a try (without the whole setup) you can use the cloud platform for free: https://budibase.com/ - then self-host later if you like it.

Happy to give some more help to self-host also.

**Comment by enpro-github** [comment]
Hey thank you for reaching out! I really appreciate that. Since it’s not available on Windows Server 2016 it seems my only option is to cloud host, I also have Linux installed on an AWS EC2 instance I guess that’s an option too?

If not then cloud hosting it is

**Reply by mjashanks** [reply]
Yes - most Linux will work. AWS EC2 will be perfect.

**Comment by enpro-github** [comment]
Okay cool, and the steps would be the same as the ones outlined on the website?

**Reply by mjashanks** [reply]
yes

**Comment by enpro-github** [comment]
Hey how easy is it to update budibase? I’m currently trying to update it (from 0.9.173) in my website I’m hosting but it’s telling me it’s up to date. Even though the latest release is 1.0.35

Thank you

**Reply by shogunpurple** [reply]
Hi @enpro-github - just to check, is your watchtower service currently running?

what's the output of `docker ps` on your server?

**Comment by enpro-github** [comment]
Hmm I don’t think it’s listed here, but the output is:

CONTAINER ID   IMAGE                                      COMMAND                  CREATED        STATUS                PORTS                                                           NAMES
36dd383db452   envoyproxy/envoy:v1.16-latest              "/docker-entrypoint.…"   2 months ago   Up 2 days             0.0.0.0:80->10000/tcp, :::80->10000/tcp                         budibase-proxy-service-1
74f9fae0a97a   budibase.docker.scarf.sh/budibase/apps     "docker-entrypoint.s…"   2 months ago   Up 2 days             4001/tcp, 0.0.0.0:4002->4002/tcp, :::4002->4002/tcp             bbapps
dbc034229c8e   budibase.docker.scarf.sh/budibase/worker   "docker-entrypoint.s…"   2 months ago   Up 2 days             4001/tcp, 0.0.0.0:4003->4003/tcp, :::4003->4003/tcp             bbworker
a2768502fac7   minio/minio                                "/usr/bin/docker-ent…"   2 months ago   Up 2 days (healthy)   0.0.0.0:4004->9000/tcp, :::4004->9000/tcp                       budibase-minio-service-1
096dd8839681   redis                                      "docker-entrypoint.s…"   2 months ago   Up 2 days             0.0.0.0:6379->6379/tcp, :::6379->6379/tcp                       budibase-redis-service-1
4f14427cc355   ibmcom/couchdb3                            "/docker-entrypoint.…"   2 months ago   Up 2 days             4369/tcp, 9100/tcp, 0.0.0.0:4005->5984/tcp, :::4005->5984/tcp   budibase-couchdb-service-1

Does it need to be mounted or something?

**Comment by enpro-github** [comment]
I also tried running `budi hosting –update` on my server and I receive the following error:

failed to register layer: Error processing tar file(exit status 1): write /usr/local/share/.cache/yarn/v6/npm-jsdom-16.7.0-918ae71965424b197c819f8183a754e18977b710-integrity/node_modules/jsdom/.yarn-tarball.tgz: no space left on device

I’ve tried running some commands like “docker system prune” but to no avail

Thank you

**Comment by enpro-github** [comment]
I was able to get watchtower running

***@***.***
